### PR TITLE
Adding support to get details for TargetPrefix

### PR DESCRIPTION
**Pull request by ina-stoyanova**
This PR adds the functionality needed in this code changes: https://github.com/gruntwork-io/terragrunt/pull/1507

With this PR we can:
- get the `TargetPrefix` specified for a state bucket with enabled access logging

Issue linked: https://github.com/gruntwork-io/terragrunt/issues/1437

### CODE REVIEW

File: modules/aws/s3.go [hunk 1]
```diff
@@ -262,33 +262,34 @@ func EmptyS3BucketE(t testing.TestingT, region string, name string) error {
...
+		return "", "", err
 	}
 
 	if res.LoggingEnabled == nil {
-		return "", S3AccessLoggingNotEnabledErr{bucket, awsRegion}
+		return "", "", S3AccessLoggingNotEnabledErr{bucket, awsRegion}
 	}
 
-	return aws.StringValue(res.LoggingEnabled.TargetBucket), nil
+	return aws.StringValue(res.LoggingEnabled.TargetBucket), aws.StringValue(res.LoggingEnabled.TargetPrefix), nil
```
Comment (brikis98) [inline modules/aws/s3.go:292]: To maintain backwards compatibility, should there be a separate method to return just the target prefix? Or possibly the bucket + prefix?

Comment (infraredgirl) [reply modules/aws/s3.go:292]: +1 to introduce a separate method that just retrieves the target prefix.

Comment (ina-stoyanova) [reply modules/aws/s3.go:292]: That's a really good point! Thanks a lot both 🙂

**Comment by ina-stoyanova** [comment]
Took me a while in the midst of support to come back here _(I thought it would require a lot more changes, but turns out it was this quick)._

Another look would be very appreciated!

File: modules/aws/s3.go (hunk 1 above)
Comment (ina-stoyanova) [reply modules/aws/s3.go]: So in terms of backwards compatibility - did you also mean to keep the naming of the method as it was originally? It was quite a generic name, so there's now 2 functions: `GetS3BucketLoggingTarget` & `GetS3BucketLoggingTargetPrefix`

Comment (brikis98) [reply modules/aws/s3.go]: Yea, I think preserving the old name makes sense too. It's not an ideal name, but probably not bad enough it merits a backwards incompatible change.

Comment (ina-stoyanova) [reply modules/aws/s3.go]: Ok, will change that too - thanks, Jim!

**Review by brikis98 — APPROVED** [review]
LGTM!

**Comment by ina-stoyanova** [comment]
Thanks Jim!

**Pull request merged**